examples/gno.land/p/demo/int256/cmp.gno

```diff
@@ -0,0 +1,86 @@
+package int256
+
+// Eq returns true if z == x
+func (z *Int) Eq(x *Int) bool {
+	return (z.neg == x.neg) && z.abs.Eq(x.abs)
+}
+
+// Neq returns true if z != x
+func (z *Int) Neq(x *Int) bool {
+	return !z.Eq(x)
+}
```

Comment:
There are missing tests for this functionality

Reply:
c890dfba0c826fedfab64a96a4ecc7ddf8469df6